subsys/bluetooth/host/conn.c

```diff
@@ -785,7 +794,7 @@ static int conn_prepare_events(struct bt_conn *conn,
 		k_poll_event_init(&events[0],
 				  K_POLL_TYPE_SEM_AVAILABLE,
 				  K_POLL_MODE_NOTIFY_ONLY,
-				  bt_conn_get_pkts(conn));
+				  conn_pkts);
```

Comment:
Is it ok to pass (potentially) NULL here?

Reply:
Oh, good point. I had read `!buffers_available` as `buffers_available` and figured that `conn_pkts` was never NULL here, but it can be. 

This would throw an assert, and needs to be fixed

Reply:
Fixed in line 786